packages/vendure-plugin-customer-managed-groups/test/customer-managed-groups.spec.ts

```diff
@@ -277,6 +279,168 @@ describe('Customer managed groups', function () {
     expect(marques).toBeUndefined();
   });
 
+  it('Members can update their own profiles with this query', async () => {
+    await authorizeAsGroupParticipant();
+    const { myCustomerManagedGroup: group } = await shopClient.query(
+      myCustomerManagedGroupQuery
+    );
+    const authorizedCustomer = group.customers.find(
+      (c: any) => c.emailAddress === 'eliezer56@yahoo.com'
+    );
+    const { updateCustomerManagedGroupMember: newGroup } =
+      await shopClient.query(updateCustomerManagedGroupMemberMutation, {
+        input: {
+          lastName: 'Teklu',
+          customerId: authorizedCustomer.customerId,
+        },
+      });
+    const authorizedCustomerUpdated = newGroup.customers.find(
+      (c: any) => c.emailAddress === 'eliezer56@yahoo.com'
+    );
+    expect(authorizedCustomerUpdated.lastName).toBe('Teklu');
+  });
+
+  it('Members can not update any other profiles ', async () => {
+    await authorizeAsGroupParticipant();
+    const { myCustomerManagedGroup: group } = await shopClient.query(
+      myCustomerManagedGroupQuery
+    );
+    const unAuthorizedCustomer = group.customers.find(
+      (c: any) => c.emailAddress === 'hayden.zieme12@hotmail.com'
+    );
+    try {
+      const { updateCustomerManagedGroupMember: newGroup } =
+        await shopClient.query(updateCustomerManagedGroupMemberMutation, {
+          input: {
+            lastName: 'Teklu',
+            customerId: unAuthorizedCustomer.customerId,
+          },
+        });
+    } catch (e) {
+      expect((e as any).response.errors[0].message).toBe(
+        `You are not allowed to update other member's details`
+      );
+    }
+  });
+  it('Unauthenticated users can not update any profiles', async () => {
+    try {
```

Comment:
You can do `await shopclient.asAnonymousUser()` to log out. After that the test should check for an `ForbiddenError` instead of our custom message '`You are not allowed to update other member's details`'